packages/pocket-ic/src/common/rest.rs

```diff
@@ -578,20 +578,7 @@ pub struct IcpFeatures {
     pub nns_governance: Option<EmptyConfig>,
     pub sns: Option<EmptyConfig>,
     pub ii: Option<EmptyConfig>,
-}
-
-impl IcpFeatures {
-    pub fn all_icp_features() -> Self {
```

Comment:
Can we keep this? Maybe just rename to something like 

`core_features`

and add a doc string to say this enables everything that doesn't require auto progressing.

Reply:
Would it be ok to add it in a separate PR _after_ the public release that I want to cut tonight? It is easy for everybody to define such a (private) helper and I'd prefer to give it a bit more thought before we add a public one.

Reply:
Agreed to rethink this outside of this PR